node/tests/k8st/tests/test_simple.py

```diff
@@ -123,136 +123,3 @@ def test_default_pods_running(self):
     def test_calico_monitoring_pods_running(self):
         with DiagsCollector():
             self.check_pod_status('calico-monitoring')
-
-
-class TestSimplePolicy(TestBase):
-    def setUp(self):
-        TestBase.setUp(self)
-        self.create_namespace("policy-demo")
-        self.deploy(NGINX_IMAGE, "nginx", "policy-demo", 80)
-
-        # Create two client pods that live for the duration of the
-        # test.  We will use 'kubectl exec' to try wgets from these at
-        # particular times.
-        #
-        # We do it this way - instead of one-shot pods that are
-        # created, try wget, and then exit - because it takes a
-        # relatively long time (7 seconds?) in this test setup for
-        # Calico routing and policy to be set up correctly for a newly
-        # created pod.  In particular it's possible that connection
-        # from a just-created pod will fail because that pod's IP has
-        # not yet propagated to the IP set for the ingress policy on
-        # the server pod - which can confuse test code that is
-        # expecting connection failure for some other reason.
-        kubectl("run access -n policy-demo" +
```

Comment:
Remove this test for a couple of reasons:

- It deploys a pod with floatingIPs, which isn't supported in the operator.
- However, it doesn't actually use that floating IP (so we're not losing test coverage of it)
- This is a duplicate of tests that are run in the e2e CI target.